lib/pact/consumer/mock_service/web_request_administration.rb

```diff
@@ -0,0 +1,30 @@
+require 'pact/consumer/mock_service/rack_request_helper'
+require 'pact/consumer/mock_service/mock_service_administration_endpoint'
+
+module Pact
+  module Consumer
+
+    # Administration web requests (GET, DELETE, POST, PUT, etc.)
+    class WebRequestAdministration < MockServiceAdministrationEndpoint
```

Comment:
Having a hard time with names.

I've subclassed the mock requests into:
- web request options for all the OPTIONS calls. (Pact ones plus the extra ones for the user).
- web request administration for the rest of the pact stuff.

Why web request? Because the code is handling what is sent by the browser. Happy to take any better name.